<launch>
  <node name="detect_grasps_server" pkg="gpd_ros" type="detect_grasps_server" output="screen">

    <param name="camera_position" value="0 0 0"/>
    <!-- What type of point cloud is used and what ROS topic it comes from -->
    <param name="cloud_type" value="0" /> <!-- 0: PointCloud2, 1: CloudIndexed, 2: CloudSamples -->
    <param name="cloud_topic" value="camera/depth/points" />
    <param name="rviz_topic" value="gpd_visual"/>
    <param name="workspace" value="0 0 0 0 0 0"/>

    <!-- (optional) The ROS topic that the samples come from (default: an empty string) -->
    <param name="samples_topic" value="" />

    <!-- Filepath to the configuration file for GPD -->
    <!-- <param name="config_file" value="/home/atmaraaj/hrg/gpd/cfg/ros_eigen_params.cfg" /> -->
    <!-- <param name="config_file" value="/home/ur5/projects/gpd/cfg/ros_vino_params.cfg" /> -->
    <param name="config_file" value="/home/shambo/tools/gpd/cfg/eigen_params.cfg" />

    <param name="rviz_topic" value="plot_grasps" />

  </node>

</launch>
